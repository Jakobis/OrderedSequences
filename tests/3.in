50
a 1859363540
a 427890845
a 654671267
su 654671267
su 654671267
s 2
a 307843486
a 1543166661
s 3
s 1
r 654671267
a 1039116110
s 4
s 0
sum
su 427890845
sum
s 5
r 1859363540
sum
s 1
pr 654671267
sum
su 1039116110
s 4
a 1888351048
pr 1039116110
d 1
a 977311016
r 1543166661
re 1859363540
r 654671267
su 1543166661
r 307843486
r 977311016
su 1543166661
a 1228575610
su 307843486
sum
a 271671331
a 1620650861
a 1243460288
a 874273387
a 38555200
su 654671267
a 1824430319
a 1549123325
a 1091786187
s 3
r 1228575610
