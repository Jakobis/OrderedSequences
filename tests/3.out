1859363540
1859363540
1859363540
1543166661
427890845
2
1543166661
307843486
5832051909
654671267
5832051909
1859363540
5
5832051909
427890845
427890845
5832051909
1543166661
1543166661
654671267
4
1
1888351048
0
2
1888351048
654671267
7639035198
874273387
654671267
8
